src/app/homework/modals/CreateHomeworkModal.js

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import '../../commons/modals/TextareaModal.scss'
+import {createRef, useState} from "react";
+import {renderModal} from "../../commons/modals/modal";
+import {ModalInput} from "../../commons/modals/ModalInput";
+import "../../homework/modals/CreateHomeworkModel.css";
+import {ModalHeader} from "../../commons/modals/ModalHeader";
+
+
+const CreateHomeworkModal = ({show, onClose}) => {
+    const [name, setName] = useState(null);
```

Comment:
const [name, setName] = useState(undefined);

Reply:
Prefer `undefined` than `null`, the former is the default init value.